src/DocumenterLaTeX.jl

```diff
@@ -2,6 +2,8 @@ module DocumenterLaTeX
 
 using Documenter
 
+const LaTeX = Documenter.Writers.LaTeXWriter.LaTeX
```

Comment:
Any thoughts on whether we should have this be exported? `makedocs(format = LaTeX())` looks nicer than `makedocs(format=DocumenterLatex.LaTeX()`.

Reply:
I agree, the annoyance is that we can not expor HTML since base exports that, so would be a bit unsymmetric. We also can't export Markdown from that PR so 🤷‍♂️

Reply:
I would be fine with HTML being a bit special. We could also rename it.. although I do not have any ideas at the moment.

Exporting `DocumenterMarkdown.Markdown` would be fine though, no?

Reply:
But then people can't use the Markdown stdlib, but maybe thats fine.

Reply:
Actually, if someone wants to use the stdlib they can 
```
import DocumenterMarkdown
```
instead.

Reply:
Yup, and most `make.jl` scripts will not `import Markdown` I reckon.